### PR TITLE
Add some translation in Kab

### PR DESCRIPTION
**Pull request by VoXaN24**

### CODE REVIEW

File: locales/kab_DZ/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -71,20 +71,7 @@ msgstr "%s deg %s"
...
-"%sTekki %sAllow%sTekki %sAdd%sSenqed %sAllow%s, sakin tekki ɣef %sOkay%s"
-
-msgctxt "install-duckduckgo"
-msgid ""
-"%sClick %sContinue To Installation%sClick %sAdd%sCheck %sAllow%s, then click "
-"%sOkay%s"
-msgstr ""
-"%sTekki %sKemmel tasuqqilt%sTekki %sAdd%sSennqed %sAllow%s, sakin tekki ɣef "
-"%sOkay%s"
+msgstr "%sKliki %s"Rnu%sYafhas %sSireg%s, sakin kiliki %sOk%s"
```
Comment (GioSensation) [inline locales/kab_DZ/LC_MESSAGES/duckduckgo.po:74]: It looks like you accidentally deleted a bunch of strings, plus there is a problem with that `%s"Rnu%s`. The `"` closes the string and creates a syntax error. Do you mind reviewing and getting back to me?

**Comment by GioSensation** [comment]
Hey @VoXaN24. It looks like there is a syntax error and you have accidentally deleted some strings. Do you mind taking another look?

File: locales/kab_DZ/LC_MESSAGES/duckduckgo.po (hunk 1 above)
Comment (VoXaN24) [reply locales/kab_DZ/LC_MESSAGES/duckduckgo.po:74]: I do it asap

**Comment by VoXaN24** [comment]
Normaly it's ok now

File: locales/kab_DZ/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -68,9 +68,6 @@ msgstr "%s mi"
 msgctxt "maps_places"
 msgid "%s on %s"
 msgstr "%s deg %s"
-
-msgctxt "install-duckduckgo"
-msgid "%sClick %sAdd%sCheck %sAllow%s, then click %sOkay%s"
 msgstr ""
```
Comment (GioSensation) [inline locales/kab_DZ/LC_MESSAGES/duckduckgo.po:71]: Moving this back to its correct place and fixing the syntax error.

```suggestion

msgctxt "install-duckduckgo"
msgid "%sClick %sAdd%sCheck %sAllow%s, then click %sOkay%s"
msgstr "%sKliki %sRnu%sYafhas %sSireg%s, sakin kiliki %sOk%s"
```

File: locales/kab_DZ/LC_MESSAGES/duckduckgo.po [hunk 3]
```diff
@@ -84,7 +81,11 @@ msgid ""
 "%sOkay%s"
 msgstr ""
 "%sTekki %sKemmel tasuqqilt%sTekki %sAdd%sSennqed %sAllow%s, sakin tekki ɣef "
-"%sOkay%s"
+"%sOkay%s
+
+msgctxt "install-duckduckgo"
+msgid "%sClick %sAdd%sCheck %sAllow%s, then click %sOkay%s"
+msgstr "%sKliki %sRnu%sYafhas %sSireg%s, sakin kiliki %sOk%s"
```
Comment (GioSensation) [inline locales/kab_DZ/LC_MESSAGES/duckduckgo.po:88]: Moving the string back to its place.

```suggestion
"%sOkay%s
```

File: locales/kab_DZ/LC_MESSAGES/duckduckgo.po [hunk 4]
```diff
@@ -208,7 +208,7 @@ msgstr "Fell-aɣ"
 #. This is an option for a user setting. It refers to showing the URL for a result ABOVE the TITLE of the result -- Example image: https://i.imgur.com/bTm8PCc.png
 msgctxt "settingsvalue"
 msgid "Above Title"
-msgstr ""
+msgstr "azwel unnig"
```
Comment (GioSensation) [inline locales/kab_DZ/LC_MESSAGES/duckduckgo.po:211]: Adding capitalization.

```suggestion
msgstr "Azwel Unnig"
```

File: locales/kab_DZ/LC_MESSAGES/duckduckgo.po [hunk 5]
```diff
@@ -104,15 +104,15 @@ msgstr "%sIssin ugar%s."
 
 msgctxt "precise_user_location"
 msgid "%sLearn how we keep your location private%s."
-msgstr ""
+msgstr "%sSselmed amek aneɣ ɛass nwen ɣben"
```
Comment (GioSensation) [inline locales/kab_DZ/LC_MESSAGES/duckduckgo.po:107]:
```suggestion
msgstr "%sSselmed amek aneɣ ɛass nwen ɣben%s"
```

**Comment by GioSensation** [comment]
Had to fix a couple more things, but now this is good to go. Thanks for your contribution. Merged! 🎉

**Pull request merged**